convert2rhel/unit_tests/subscription_test.py

```diff
@@ -293,12 +293,43 @@ def test_remove_original_subscription_manager(self):
 
         self.assertEqual(utils.remove_pkgs.called, 1)
 
+    @unit_tests.mock(pkghandler, "call_yum_cmd", lambda a, b, enable_repos, disable_repos, set_releasever: (None, 0))
+    @unit_tests.mock(logging.Logger, "info", LogMocked())
     @unit_tests.mock(os.path, "isdir", lambda x: True)
     @unit_tests.mock(os, "listdir", lambda x: ["filename"])
+    @unit_tests.mock(subscription, "check_for_pkg_presence", lambda x: ["test"])
+    @unit_tests.mock(subscription, "get_installed_submgr_pkg_names", lambda x: ["test"])
+    @unit_tests.mock(utils.changed_pkgs_control, "track_installed_pkgs", lambda x: None)
+    def test_install_rhel_subscription_manager(self):
+        subscription.install_rhel_subscription_manager()
+        self.assertTrue("\nPackages installed:\n" in logging.Logger.info.msg)
```

Comment:
No assertions are being made about if packages get untracked from the tracked packages in `utils`. `check_for_pkg_presence` could return correct packages but we don't make any assertions that the return value in `rpms_to_untrack` is being used. Same applies to `pkg_names`

If I were to do this I'd expect a test to fail
```python
# rpms_to_untrack = check_for_pkg_presence(pkg_names)
rpms_to_untrack = []
# or 
rpms_to_untrack = ["should-fail"]
```

Reply:
You're right! Actually, this reminds me that the `rpms_to_untrack` was not being asserted on the code either!

Reply:
I've updated the test with some assertions, but don't know if they are the right ones. I believe with the assertations I just updated, we can validate that the `rpms_to_untrack` was rightfully called?